src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs

```diff
@@ -50,7 +50,7 @@ public string GetCurrentRequestHeader(ApiEvent eventInProgress)
             sb.Append(eventInProgress.IsLegacyCacheEnabledString);
             sb.Append(TelemetryConstants.CommaDelimiter);
             // Token type is to indicate 1 - bearer, 2 - pop, 3 - ssh-cert, 4 - external.
```

Comment:
Add 5 - extension to this comment?

Reply:
maintenance hazard, is that comment needed?

Reply:
4 is not external, 4 is "AT POP"

Reply:
Where do we use "AT POP"? i dont see it anywhere in the codebase as a token type.

Reply:
it is used to implement what we call in our codebase "Legacy POP". It's an SHR POP used internally. Very similar to the regular POP used by public client.